src/screens/Profile/Profile.js

```diff
@@ -330,12 +365,37 @@ class Profile extends React.Component<Props, State> {
       showSystemInfoModal,
       showCheckPinModal,
       showBiometricsSelector,
+      showTrackingModal,
     } = this.state;
 
     const isWalletBackedUp = isImported || isBackedUp;
     return (
       <Container inset={{ bottom: 0 }}>
         <Header gray title="settings" onBack={() => navigation.goBack(null)} />
+        <SlideModal
+          isVisible={showTrackingModal}
+          fullScreen
+          showHeader
+          onModalHide={this.toggleTrackingModal}
+          backgroundColor={baseColors.lightGray}
+          avoidKeyboard
+        >
+          <Wrapper regularPadding flex={1}>
+            <SettingsModalTitle extraHorizontalSpacing>
```

Comment:
I believe we do not need extra horizontal spacing here - header gets shifted to right too much compared to other similar headers on this screen's modals. Unless its been decided otherwise :)

Reply:
You're right, it looks weird. I've updated it and it looks a bit better now:

![tracking-3](https://user-images.githubusercontent.com/263335/60203559-46c36a80-9823-11e9-8b13-9981e428a7b2.png)

Reply:
Great! :)